### PR TITLE
DTSPB-2886 Scheduler job for move in and out of Dormant

### PR DESCRIPTION
**Pull request by IswaryaPepakayala**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-2800
https://tools.hmcts.net/jira/browse/DTSPB-2886

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by sanjayparekh-hmcts — APPROVED** [review]

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+spec:
+  releaseName: probate-cron-make-dormant-cases
+  chart:
+    git: git@github.com:hmcts/probate-cron
+    ref: master
+    path: probate-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/probate/orchestrator-service:prod-fbc4ab7-20220526135216 #{"$imagepolicy": "flux-system:probate-orchestrator-service"}
+      keyVaults:
```
Comment (srochani) [inline k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml:14]: Remove these all secrets

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/demo-00.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      cpuLimits: "2000m"
+      image: hmctspublic.azurecr.io/probate/back-office:pr-2018-287b2bd-20220815202906 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
+      environment:
+        CCD_GATEWAY_HOST: https://manage-case.demo.platform.hmcts.net
+        DUMMY_PROPERTY: DUMMY8
+        PROBATE_POSTGRESQL_USER: probateman_user@probatemandb-postgres-db-v11-demo
+        PROBATE_POSTGRESQL_HOSTNAME: probatemandb-postgres-db-v11-demo.postgres.database.azure.com
+        EXCEPTED_ESTATE_DATE_OF_DEATH: 2022-01-01
+        IDAM_SERVICE_HOST: https://idam-api.demo.platform.hmcts.net
+      keyVaults:
```
Comment (srochani) [inline k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/demo-00.yaml:26]: Remove these secrets

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+spec:
+  releaseName: probate-cron-reactivate-dormant-cases
+  chart:
+    git: git@github.com:hmcts/probate-cron
+    ref: master
+    path: probate-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/probate/orchestrator-service:prod-fbc4ab7-20220526135216 #{"$imagepolicy": "flux-system:probate-orchestrator-service"}
+      keyVaults:
```
Comment (srochani) [inline k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml:14]: Remove these secrets

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  name: probate-cron-make-dormant-cases
+spec:
+  releaseName: probate-cron-make-dormant-cases
+  chart:
+    git: git@github.com:hmcts/probate-cron
+    ref: master
+    path: probate-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/probate/orchestrator-service:prod-fbc4ab7-20220526135216 #{"$imagepolicy": "flux-system:probate-orchestrator-service"}
```
Comment (srochani) [inline k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml:13]: This should refer to BO image

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  name: probate-cron-reactivate-dormant-cases
+spec:
+  releaseName: probate-cron-reactivate-dormant-cases
+  chart:
+    git: git@github.com:hmcts/probate-cron
+    ref: master
+    path: probate-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/probate/orchestrator-service:prod-fbc4ab7-20220526135216 #{"$imagepolicy": "flux-system:probate-orchestrator-service"}
```
Comment (srochani) [inline k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml:13]: this should refer to BO image

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml (hunk 1 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml]: Removed

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/demo-00.yaml (hunk 2 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/demo-00.yaml]: Removed and modified the name

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml (hunk 3 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml]: Removed

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml (hunk 5 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml]: Changed

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  name: probate-cron-reactivate-dormant-cases
+spec:
+  releaseName: probate-cron-reactivate-dormant-cases
+  chart:
+    git: git@github.com:hmcts/probate-cron
+    ref: master
+    path: probate-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/probate/back-office:prod-7c60114-20220816141606 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
```
Comment (srochani) [inline k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml:13]: please remove the demo- from image policy

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  name: probate-cron-make-dormant-cases
+spec:
+  releaseName: probate-cron-make-dormant-cases
+  chart:
+    git: git@github.com:hmcts/probate-cron
+    ref: master
+    path: probate-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/probate/back-office:prod-7c60114-20220816141606 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
```
Comment (srochani) [inline k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml:13]: please remove the demo- from image policy

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml (hunk 6 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml]: Removed

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml (hunk 7 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml]: Removed

**Review by sanjayparekh-hmcts — APPROVED** [review]

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/aat-00.yaml [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: probate-cron-make-dormant-cases
+spec:
+  releaseName: probate-cron-make-dormant-cases
+  values:
+    global:
+      jobKind: CronJob
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: aat
```
Comment (DelythJustice) [inline k8s/namespaces/probate/probate-cron-make-dormant-cases/aat-00.yaml:12]: this is missing job schedule

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/demo.yaml [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: demo
+    job:
+      memoryRequests: "1024Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "2048Mi"
+      cpuLimits: "2000m"
+      image: hmctspublic.azurecr.io/probate/back-office:pr-2018-f73d655-20220817104223 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
+      schedule: 0/10 * * * *
```
Comment (DelythJustice) [inline k8s/namespaces/probate/probate-cron-make-dormant-cases/demo.yaml:19]: do you want this to run every 10 minutes in demo?

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+  chart:
+    git: git@github.com:hmcts/probate-cron
+    ref: master
+    path: probate-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/probate/back-office:prod-6e7eee4-20220818132310 #{"$imagepolicy": "flux-system:probate-back-office"}
+      environment:
+        TASK_NAME: MakeDormantCasesTask
+      schedule: 0/10 * * * *
```
Comment (DelythJustice) [inline k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml:16]: do you want this to run every 10minutes?

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/demo.yaml [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: demo
+    job:
+      memoryRequests: "1024Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "2048Mi"
+      cpuLimits: "2000m"
+      image: hmctspublic.azurecr.io/probate/back-office:pr-2018-f73d655-20220817104223 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
+      schedule: 0/10 * * * *
```
Comment (DelythJustice) [inline k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/demo.yaml:19]: and this every 10minutes?

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+  chart:
+    git: git@github.com:hmcts/probate-cron
+    ref: master
+    path: probate-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/probate/back-office:prod-6e7eee4-20220818132310 #{"$imagepolicy": "flux-system:probate-back-office"}
+      environment:
+        TASK_NAME: ReactivateDormantCasesTask
+      schedule: 0/10 * * * *
```
Comment (DelythJustice) [inline k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml:16]: same

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/demo.yaml (hunk 9 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-make-dormant-cases/demo.yaml:19]: For testing purpose I have added 10 mins as suggested by Sunil

File: k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml (hunk 10 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-make-dormant-cases/probate-cron-make-dormant-cases.yaml:16]: For testing purpose I have added

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/demo.yaml (hunk 11 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/demo.yaml:19]: For testing purpose I have added

File: k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml (hunk 12 above)
Comment (IswaryaPepakayala) [reply k8s/namespaces/probate/probate-cron-reactivate-dormant-cases/probate-cron-reactivate-dormant-cases.yaml:16]: For testing purpose I have added

**Review by DelythJustice — APPROVED** [review]

**Pull request closed without merging**